php-checks/src/main/java/org/sonar/php/checks/formatting/IndentationCheck.java

```diff
@@ -76,17 +76,17 @@ private void checkArgumentsIndentation(FormattingStandardCheck formattingCheck,
       int expectedIndentationColumn = getLineStartingColumn(node.getParent()) + PSR2_INDENTATION;
       int callingLine = methodName.getTokenLine();
 
-      if (!isOnSameLine(methodName.getToken(), lastParam)) {
+//      if (!isOnSameLine(methodName.getToken(), lastParam)) {
 
         if (!isCorrectlySplittedOnLines(callingLine, arguments)) {
-          formattingCheck.reportIssue("Either split this list into multiple lines, aligned at column \""
-            + expectedIndentationColumn + "\" or put all arguments on line \""
-            + callingLine + "\".", firstParam);
+//          formattingCheck.reportIssue("Either split this list into multiple lines, aligned at column \""
+//            + expectedIndentationColumn + "\" or put all arguments on line \""
+//            + callingLine + "\".", firstParam);
         } else if (!isCorrectlyIndented(expectedIndentationColumn, arguments)) {
-          formattingCheck.reportIssue("Align all arguments in this list at column \"" + expectedIndentationColumn + "\".", firstParam);
+//          formattingCheck.reportIssue("Align all arguments in this list at column \"" + expectedIndentationColumn + "\".", firstParam);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)